app/scripts/models/order.coffee

```diff
@@ -0,0 +1,84 @@
+root = exports ? this
+
+unt = require './unit'
+
+# TODO(cosmic): Locations need a class for normalization.
+root.Order = class Order
+  constructor: (order) ->
+    # Actor unit.
+    @unit = order.unit
+    # Action.
+    @act = order.act
```

Comment:
I tend to err on the side of spelling out names instead of abbreviating. `@action` leaves much less doubt about its intent than `@act`.